### DISCUSSION TITLE
Copilot works locally but not on SSH

### DISCUSSION THREAD

**Discussion by elchorro** [community/community#35125] [Copilot Conversations]
So I have been successfully using Copilot through VSCode for a while but sadly it does not get activated when I log in through SSH. Consequentely I don't see any COpilot comands in the Palette.

The extension shows _Not yet activated_ under Runtime Status
![image](https://user-images.githubusercontent.com/13162607/193940859-230983d4-1389-453d-a716-766d9680d17d.png)

I tried several times to log out and log in to my github account and to reload the window. To no avail. 

Do you have any ideas what to do?

**Comment by emploiyer** [comment]
Non pas vraiment
